lectures/tools_and_techniques/stationary_densities.md

```diff
@@ -462,18 +462,18 @@ Random.seed!(42) # For deterministic results.
 
 s = 0.2
 δ = 0.1
```

Comment:
`delta` to list

Reply:
Great.  I think for these things you can just do "resolve conversation" and when I do the review I will see them.